README.md

```diff
@@ -1 +1,228 @@
+
+## 🛠 Built With <a name="React"> React and Redux, JavaScript</a>
+
+### Tech Stack <a name="tech-stack">Computer and Code editor</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Rockets]**
+- **[Mission]**
+- **[My profile]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://strong-heliotrope-adde83.netlify.app)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- a computer and a dode Editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Setup Linters by running following commands in your terminal:
+
+- npm install --save-dev stylelint@13.x stylelint-scss@3.x - - - stylelint-config-standard@21.x stylelint-csstree-validator@1.x (Install the StyleHint linters)
+
+- npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
+
+- npx stylelint "*/.{css,scss}" (checks for the CSS related errors in your project)
+
+- npx stylelint "*/.{css,scss}" --fix (This will fix the linters error, if any)
+
+- npm install --save-dev hint@7.x (Install the WebHint Linters)
+
+- Run npx eslint . on the root of your directory of your project.
+
+
+
+### Install
+
+Install this project with:
+
+- use npm create-react-app to have a react template
+- use npm instapll react-redux toolkit to use redux in react.
+
+### Usage
+
+To run the project, execute the following command:
+
+- run npm start to run the project on the UI
+
+### Run tests
+
+To run tests, run the following command:
+
+- Run npm test to run tests
+
+### Deployment
+
+You can deploy this project using:
+-  Netlify Dashboard.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@Irirwanirira](https://github.com/Irirwanirira)
+- Twitter: [@Iri_joseph](https://twitter.com/Irirwanirira)
+- LinkedIn: [Joseph IRIRWANIRIRA](https://linkedin.com/in/joseph-irirwanirira-74666623a/)
+
+👤 **Author2**
+
+- GitHub: @jaamanu
+- Twitter: @JamesAsibeyManu
+- LinkedIn: @jamesasibeymanu
+
```

Comment:
- [ ] To make your README more professional and `## Authors` section matches to the one of Microverse template, kindly make sure you update it by replacing where it is written `👤 **Author...**` with `👤 **Author's full name**` and for the second `author` to add his social links adreeses as you did for the **1st author**